style.css

```diff
@@ -0,0 +1,87 @@
+html {
+    box-sizing: border-box;
+    background: #ffc600;
+    font-family: 'helvetica neue';
+    font-size: 20px;
+    font-weight: 200;
+}
+  
+body {
+    margin: 0;
+}
+  
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+.panels {
+    min-height: 100vh;
+    overflow: hidden;
+    display: flex;
+}
+
+.panel {
+    background: #6B0F9C;
+    box-shadow: inset 0 0 0 5px rgba(255,255,255,0.1);
+    color: white;
+    text-align: center;
+    transition: font-size 0.7s cubic-bezier(0.67, -0.23, 1, -0.35),
+     flex 0.7s cubic-bezier(0.61,-0.19, 0.7,-0.11);
+    font-size: 20px;
+    background-size: cover;
+    background-position: center;
+    flex: 1;
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+}
+
+.panel1 {background-image:url(./jpg/1.jpg);}
+.panel2 {background-image:url(./jpg/2.jpg);}
+.panel3 {background-image:url(./jpg/3.jpg);}
+.panel4 {background-image:url(./jpg/4.jpg);}
+.panel5 {background-image:url(./jpg/5.jpg);}
+
+/* Flex Children */
+.panel > * {
+    margin: 0;
+    width: 100%;
+    transition: transform 0.5s;
+    flex: 1 0 auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.panel > :nth-child(1) {
+    transform: translateY(-100%);
+}
+
+.panel.open-active > :nth-child(1){
+    transform: translateY(0);
+}
+
+.panel > :nth-child(3) {
+    transform: translateY(100%);
+}  
+
+.panel.open-active > :nth-child(3) {
+    transform: translateY(0);
+} 
+
+.panel p {
+    text-transform: uppercase;
+    font-family: 'Amatic SC', cursive;
+    text-shadow: 0 0 4px rgba(0, 0, 0, 0.72), 0 0 14px rgba(0, 0, 0, 0.45);
+    font-size: 2em;
```

Comment:
it would be better if the dimensions on the page were specified in the same units

Reply:
done